Child Minor Addendum.md

```diff
@@ -0,0 +1,34 @@
+**<h1>CHILD MINOR ADDENDUM</h1>**
+
+<p>This Child Minor Addendum (this <strong>“Addendum”</strong>) is between Sage, the undersigned minor child (<strong>“Minor”</strong>), and the undersigned minor child’s parent or legal guardian (<strong>“Parent”</strong>).This Addendum is incorporated into and made a part of the Sage Terms, of Service (<strong>“Terms”</strong>) effective as of the date signed below. All capitalized terms used in this Addendum, but not defined, shall have the definition set forth in the Terms. In the event of any conflict between the Terms and this Addendum, this Addendum prevails.</p> 
+
+<p>By using the Service, Minor and Parent agree to the following:</p>
+
+<ol>
+  <li><p>Minor represents that it is at least 13 years old and has read and agrees to the Terms. Minor further agrees that its access to the Service is solely for the following limited purpose (DESCRIBE PURPOSE FOR ACCESS):</p></li>
+
+<li><p>Parent represents that it is the parent or legal guardian of Minor. Parent hereby represents that it has read and agrees to the Terms on behalf of itself and Minor. Parent acknowledges that Minor’s permission to use the Service is conditional upon Parent’s own, additional agreement to the Terms.</p></li>
+
+<li><p>Parent represents that it will ensure Minor’s compliance with the Terms and this Addendum. Parent assumes all liability relating to or arising from Minor’s use of the Service. Parent further agrees to defend and indemnify the Sage Entities from and against every claim brought by a third party, and any related liability, damage, loss, and expense, including attorneys’ fees and costs, arising out of or connected with: (a) Minor’s unauthorized use of, or misuse of, the Service; (b) Minor’s violation of any portion of the Terms, any representation, warranty, or agreement referenced in the Terms, or any applicable law or regulation; (c) Minor’s violation of any third-party right, including any intellectual property right or publicity, confidentiality, privacy, or other property right; or (d) any dispute or issue between Minor and any third party.</p></li>
+</li>
+</ol>
+
+<p>Sage reserves the right, at its own expense, to assume the exclusive defense and control of any matter otherwise subject to indemnification by Minor (without limiting Parent’s indemnification obligations with respect to that matter), and in that case, Parent agrees to cooperate with Sage’s defense of those claims.</p>
+
+<p><strong>Minor Signature</strong></p>
+<p>Name:</p>                                                                                                                        
+<p>Signature:</p>
+<p>Synapse User Name:</p>
+<p>Date:</p>
+
+<p><strong>Parent Signature</strong></p>
+<p>Name:</p>
+<p>Signature:</p>
+<p>email:</p>
```

Comment:
Is there a reason why "email" is lowercase, but the other labels are title case?

Reply:
No- This is a Typo